lib/matplotlib/tests/test_ticker.py

```diff
@@ -159,6 +159,49 @@ def test_SymmetricalLogLocator_set_params():
     nose.tools.assert_equal(sym.numticks, 8)
 
 
+def test_ScalarFormatter_offset_value():
```

Comment:
Awesome.  That is something that has driven me crazy a couple of times.  I have dealt with it by putting the decorator on the called function and creating all of the figures/axes in that function.